data/src/main/java/me/androidbox/data/local/dao/EventDao.kt

```diff
@@ -0,0 +1,25 @@
+package me.androidbox.data.local.dao
+
+import androidx.room.*
+import kotlinx.coroutines.flow.Flow
+import me.androidbox.data.local.DatabaseConstant
+import me.androidbox.data.local.entity.EventEntity
+
+@Dao
+interface EventDao {
+    @Query("SELECT * FROM ${DatabaseConstant.EVENT_TABLE}")
+    fun getEvent(): Flow<List<EventEntity>>
+
+    @Insert(onConflict = OnConflictStrategy.REPLACE)
+    fun insertEvent(eventEntity: EventEntity)
+
+    /** TODO
+     * Delete only a single event and all the attendees and photos that have been added to that event
+     * */
+    @Delete
+    fun deleteEvent(eventEntity: EventEntity)
```

Comment:
Is this correct, we only want to delete a single event?

Reply:
Yes, but I'd usually make this a query that only takes an event's ID and deletes based on that. You don't always have the full instance of the event before deleting it.